pywincffi/kernel32/events.py

```diff
@@ -34,7 +34,8 @@ def CreateEvent(
         A pointer to a ``SECURITY_ATTRIBUTES`` structure.  If not provided
         then by default the handle cannot be inherited by a subprocess.
 
-    :keyword str lpName:
+    :keyword unicode/str lpName:
+        Type is unicode on Python 2, str on Python 3.
```

Comment:
I'd rephrase so it's clear the doc string is talking about the objects:

```
Type is ``unicode`` on Python 2, ``str`` on Python 3.
```

You could also do:

```
Type is :class:`unicode` on Python 2, :class:`str` on Python 3.
```

I'd suggest the first version though because it render identically in Python 2 and 3 (docs could be built in either version).